microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -22,89 +22,102 @@ ms.date: 05/04/2021
 
 
 <!-- ====================================================================== -->
 ## Open resources in the Network tool
 
-See [Inspect the details of a resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_.
+See [Inspect the details of the resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_ to learn how to get the details for a specific resource.
+
 <!--why?-->
 
-<!-- need lead-in -->
+Show network resource details:
 
 ![Inspect a resource in the Network tool.](../media/resources-network-response.msft.png)
 
-### Reveal resources in the Network tool from other panels
+#### Reveal resources in the Network tool from other panels
 
 The next section, [Browse resources in the Network panel](#browse-resources-in-the-network-panel), shows how to view resources from various parts of the DevTools UI.  To inspect a resource in the **Network** tool, right-click the resource and select **Reveal in Network panel**.
 
+Pick resource to show in Network panel:
+
 ![Revealing resources in the Network panel.](../media/resources-sources-page-reveal-in-network-panel.msft.png)
 
 
 <!-- ====================================================================== -->
 ## Browse resources in the Network panel
 
 <!-- to do x, -->
+See [Log network activity](../network/index.md#log-network-activity) in the article  _Inspect network activity_ to learn how to see the network activity generated by the page resources.
 
-1. Open [Log network activity](../network/index.md#log-network-activity) (in the article  _Inspect network activity_) in a new tab or window.
+<!-- 1. Open [Log network activity](../network/index.md#log-network-activity) (in the article  _Inspect network activity_) in a new tab or window. --.
 
 <!-- 1. Do the steps in that section, and then continue below. -->
 
  <!-- open [link] in a new tab or window, and then continue below.. -->
 <!--why?-->
 
 <!-- need lead-in -->
+Show network activity for page resources:
 
 ![Page resources in the Network Log.](../media/resources-network-resources.msft.png)
 
 
 <!-- ====================================================================== -->
 ## Browse resources by directory in the Sources tool
 
-To view the resources of a webpage organized by directory:
+You can use the **Sources** tool to see webpage resources organized by directory.
+
+To use the Sources tool:
 
 1. Open DevTools.
 
 1. Select the **Sources** tool, and then in the **Navigator** pane in the upper left, select the **Page** tab.
 
 1. Click the **More options** (...) button to the right of the **Page** tab, and then select **Group by folder**.
 
+   Open Sources tool:
+
    ![The Page tab in the Navigator pane of the Sources tool.](../media/resources-sources-page-empty.msft.png)
 
     Here is a breakdown of the non-obvious items in the previous figure.
 
     | Page item | Description |
     |:--- |:--- |
```

Comment:
```suggestion
    |---|---|
```
Global: The default is simple `|---|` - make Markdown as simple/clean looking as possible.
Corroboration:
* Press Alt+M in VS Code in a .md file, then enter "table".
* https://learn.microsoft.com/en-us/contribute/markdown-reference#tables
* https://www.markdownguide.org/cheat-sheet/#extended-syntax
* https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables#creating-a-table